### PR TITLE
Add Elm golden test Json -> OneType

### PR DESCRIPTION
**Pull request by vrom911**
Part of #79

So, it's not that easy to read a file in Elm and then use it in tests (it involves HTTP queries). So I just decided to generate another Elm file which would contain the JSON string from `test/golden/oneType.json` file. It is generated each time you run `generate-elm` executable.

### CODE REVIEW

File: frontend/tests/Tests/Golden.elm [hunk 1]
```diff
@@ -0,0 +1,81 @@
+module Tests.Golden exposing (goldenOneTypeJson)
+
+goldenOneTypeJson : String
+goldenOneTypeJson =
+    """
+    {
```
Comment (chshersh) [inline frontend/tests/Tests/Golden.elm]: I want multiline strings in Haskell too :disappointed:

File: test/golden/oneType.json [hunk 2]
```diff
@@ -25,6 +25,14 @@
...
         "unit": []
     },
+    "myUnit": {
+        "tag": "MyUnit",
+        "contents": []
+    },
+    "myResult": {
+        "tag": "Err",
+        "contents": "clashing test"
+    },
```
Comment (chshersh) [inline test/golden/oneType.json]: How do you generate pretty-printed content of this file? I wonder, what should I do when I change `OneType`  definition to make tests pass...

Comment (vrom911) [reply test/golden/oneType.json]: Oh, this edit was made manually, so I think that we need a strategy to deal with such things... Do you have anything in mind?

Comment (chshersh) [reply test/golden/oneType.json]: We can use `aeson-pretty` package to create this file from `OneType` automatically. Probably as a part of `generate-elm`, but not sure...

* http://hackage.haskell.org/package/aeson-pretty

But this is definitely a separate issue

**Pull request merged**